homeassistant/components/webhook/trigger.py

```diff
@@ -62,6 +83,31 @@ async def async_attach_trigger(
 ) -> CALLBACK_TYPE:
     """Trigger based on incoming webhooks."""
     webhook_id: str = config[CONF_WEBHOOK_ID]
+    local_only = config.get(CONF_LOCAL_ONLY)
+    issue_id: str | None = None
+    if local_only is None:
+        issue_id = f"trigger_missing_local_only_{webhook_id}"
+        async_create_issue(
+            hass,
+            DOMAIN,
+            issue_id,
+            breaks_in_ha_version="2023.4.0",
+            is_fixable=False,
+            learn_more_url="https://github.com/home-assistant/core/pull/66494",
```

Comment:
Linking to a PR sounds incorrect. That is not for end-user consumption. I suggest removing the URL and providing more context in the raised repair issue instead.

Reply:
Sounds good. I removed the URL here and added a link to the public webhook automation trigger docs on the repair.